framework/source/class/qx/ui/list/List.js

```diff
@@ -597,6 +625,28 @@ qx.Class.define("qx.ui.list.List",
     },
 
 
+    /**
+     * Event handler for the updated event of the 
+     * qx.ui.virtual.layer.WidgetCell layer.
+     *
+     * Recalculates the item sizes in a deffered call,
+     * which only happens if we have variable item heights
+     */
+    _onLayerUpdated: function () {
+      if (this.isVariableItemHeight() === false) {
+        return;
+      }
+
+      if (this.__deferredLayerUpdate === null) {
+        this.__deferredLayerUpdate = new qx.util.DeferredCall(function () {
+          this._setRowItemSize();
+        }, this);
+      }
+
+      this.__deferredLayerUpdate.schedule();
+    },
+
```

Comment:
Disregard. I missed the fact that it's only creating a single DeferredCall (if deferredLayerUpdate is null). The schedule call itself only actually schedules once.

However, is there a reason not to move the schedule() call into the _if_ block so it only gets called once?